src/vivarium_csu_hypertension_sdc/components/hypertension_intervention.py

```diff
@@ -0,0 +1,159 @@
+from scipy import stats
+
+import numpy as np
+import pandas as pd
+
+from vivarium_csu_hypertension_sdc.components import utilities
+from vivarium_csu_hypertension_sdc.components.globals import (DOSAGE_COLUMNS, HYPERTENSIVE_CONTROLLED_THRESHOLD)
+
+
+class TreatmentAlgorithm:
+    configuration_defaults = {
+        'hypertension_treatment': {
+            'high_systolic_blood_pressure_measurement': {
+                'error_sd': 6,
+            },
+            'therapeutic_inertia': {
+                'mean': 0.136,
+                'sd': 0.0136,
+            },
+            'adverse_events': {
+                'mean': 0.1373333333,
+                'sd': 0.03069563849,
+            }
+        }
+    }
+
+    @property
+    def name(self):
+        return 'hypertension_treatment_algorithm'
+
+    def setup(self, builder):
+        self.clock = builder.time.clock()
+
+        self.sim_start = pd.Timestamp(**builder.configuration.time.start)
+
+        self.config = builder.configuration.hypertension_treatment
+
+        columns_created = ['followup_date', 'last_visit_date', 'last_visit_type',
+                           'high_systolic_blood_pressure_measurement',
+                           'high_systolic_blood_pressure_last_measurement_date']
+        builder.population.initializes_simulants(self.on_initialize_simulants,
+                                                 requires_columns=DOSAGE_COLUMNS,
+                                                 creates_columns=columns_created)
+        self.population_view = builder.population.get_view(DOSAGE_COLUMNS + columns_created + ['alive'],
+                                                           query='alive == "alive"')
```

Comment:
What's your preference in default queries vs. querying when you call to ``get``?  I like that this is convenient (and you don't actually need to request a view of the column to filter on it), but I don't like that the filter is far away from the usage.

Reply:
I think I prefer a default query for something like this since this component will never want a pop view that includes dead sims. It seems like it'd be too easy to accidentally forget to filter in one place and end up transitioning dead simulants (especially because I use the pop view index everywhere since the event index was giving me untracked sims)